apps/watch/src/components/VideoContentPage/VideoContentPage.tsx

```
@@ -109,17 +108,17 @@ export function VideoContentPage(): ReactElement {
               <VideoHeading
                 loading={loading}
                 hasPlayed={hasPlayed}
-                videos={realChildren}
+                videos={children}
                 onShareClick={() => setOpenShare(true)}
                 onDownloadClick={() => setOpenDownload(true)}
               />
               {((container?.childrenCount ?? 0) > 0 || childrenCount > 0) &&
-                (realChildren.length === children.length ||
-                  realChildren.length > 0) && (
+                (children.length === children.length ||
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)